documentations/linters.md

```diff
@@ -1,54 +0,0 @@
-# Linters
```

Comment:
I deleted this file because it added to our maintenance cost (e.g. the link to `alex` is incorrect). I thought it'd be better that we let the contributor search for `alex` or `markdownlint` in a different manner.